packages/wonder-blocks-data/docs.md

```diff
@@ -81,30 +81,30 @@ If the predicate is not given, all items are removed.
 ### Usage
 
 ```js static
-removeAllFromCache(predicate?: (key: string, entry: $ReadOnly<CacheEntry<TData>>) => boolean): number;
+removeAllFromCache(predicate?: (key: string, entry: $ReadOnly<CachedResponse<TData>>) => boolean): number;
 ```
 
 #### Function arguments
 
 | Argument | Flow&nbsp;Type | Default | Description |
 | --- | --- | --- | --- |
-| `predicate` | `(key: string, entry: $ReadOnly<CacheEntry<TData>>) => boolean)` | _Optional_ | A predicate to identify which entries to remove. If absent, all data is removed; if present, any entries for which the predicate returns `true` will be returned. |
+| `predicate` | `(key: string, entry: $ReadOnly<CachedResponse<TData>>) => boolean)` | _Optional_ | A predicate to identify which entries to remove. If absent, all data is removed; if present, any entries for which the predicate returns `true` will be returned. |
 
 ## Types
 
-### ResponseCache
+### CachedResponses
 
 ```js static
-type CacheEntry =
+type CachedResponse =
     | {|
         data: any,
       |}
     | {|
         error: string,
       |};
 
-type ResponseCache = {
-    [id: string]: CacheEntry,
+type CachedResponses = {
+    [id: string]: CachedResponse,
     ...,
 };
```

Comment:
I tend to avoid pluralized type names since it's hard to know if the type is an array or an object just by looking at it.  I can hover over it thought to get that info so it's not a big deal.

Reply:
Naming things is hard and with something already called `ResponseCache`, I didn't want to reuse it. 

However, we could rename that `ResponseCache` to `SsrCache` and then call this `ResponseCache`.